src/Message/Response.php

```diff
@@ -12,7 +12,11 @@ class Response extends AbstractResponse
     /**
      * @var array<int>
      */
-    protected static $SUCCESS_CODES = array(200);
+    protected static $SUCCESS_CODES = array(
```

Comment:
Do we still use the old array syntax here in order to support PHP 5.3 or does that not matter?

Reply:
yup, omnipay supports old phps